core/src/main/java/org/apache/gravitino/connector/PropertyEntry.java

```diff
@@ -64,7 +65,8 @@ private PropertyEntry(
       Function<String, T> decoder,
       Function<T, String> encoder,
       boolean hidden,
-      boolean reserved) {
+      boolean reserved,
+      boolean wildcard) {
```

Comment:
you also need to update the javaDoc on the method.

The boolean parameter name `wildcard` is not quite suitable, suggest to use  `useWildcard`, WDYT?